stdlib/2/__builtin__.pyi

```diff
@@ -1207,8 +1207,9 @@ if sys.version_info >= (3, 6):
     # This class is to be exported as PathLike from os,
     # but we define it here as _PathLike to avoid import cycle issues.
     # See https://github.com/python/typeshed/pull/991#issuecomment-288160993
-    class _PathLike(Generic[AnyStr]):
-        def __fspath__(self) -> AnyStr: ...
+    _AnyStr_co = TypeVar("_AnyStr_co", str, bytes, covariant=True)
+    class _PathLike(Protocol[_AnyStr_co]):
```

Comment:
Should we put this in `_typeshed`?

Reply:
I'd prefer not to... even if it works, it'll involve a mypy-typeshed dance (and maybe changes to other type checkers) for little benefit.
Plus _typeshed is for making types internally reusable, and PathLike is already internally (and externally) reusable from os